lib/js/_utils/_check-current-version.js

```diff
@@ -42,7 +41,7 @@ function checkCurrentVersion(packagePath) {
 		showOutput.log([{
 			type: 'info',
 			description: 'checking package',
-			message: chalk.blue(pkg.name)
+			message: pkg.name
```

Comment:
Just removing this as I noticed it wasn’t needed - this was a hangover from a previous logging style